common-arcana.lic

```diff
@@ -426,7 +426,7 @@ module DRCA
     pause 0.5 while UserVars.moons.empty?
   end
 
-  def cast_spell(data, settings, force_cambrinth = false)
+  def cast_spell(data, settings, force_cambrinth = false, location = nil)
```

Comment:
❓ How do you plan to use this? I ask because the consumer can already set `data['cast']` before calling `cast_spell`, which doesn't require a change to `common-arcana`.

Reply:
working on updating healme.lic to use waggle

Reply:
your right, we can close this. Will implement that into healme.lic instead.